### ISSUE TITLE
'spacy.tokens.token.Token object has no attribute 'strip''

### ISSUE THREAD

**Issue by scrhere82** [explosion/spaCy#5066]
import torch
from torchtext import data
from torchtext import datasets
import random
import numpy as np
import spacy
from spacy.tokenizer import Tokenizer

SEED = 1234

nlp = spacy.load("en_core_web_sm")
tokenizer = Tokenizer(nlp.vocab)

random.seed(SEED)
np.random.seed(SEED)
torch.manual_seed(SEED)
torch.backends.cudnn.deterministic = True

TEXT = data.Field(tokenize = tokenizer, batch_first = True)
LABEL = data.LabelField(dtype = torch.float)

train_data, test_data = datasets.IMDB.splits(TEXT, LABEL)

train_data, valid_data = train_data.split(random_state = random.seed(SEED))

MAX_VOCAB_SIZE = 25_000

TEXT.build_vocab(train_data, 
             max_size = MAX_VOCAB_SIZE, 
             vectors = "glove.6B.100d", 
             unk_init = torch.Tensor.normal_)

LABEL.build_vocab(train_data)
The TEXT.build_vocab is giving an error:

'spacy.tokens.token.Token' object has no attribute 'strip'.

Please help as I am stuck with it.

## Your Environment

* Operating System:        Windows-10-10.0.18362-SP0
* Python Version Used:   3.7.3
* spaCy Version Used: 2.2.3
* Environment Information:

**Comment by svlandeg** [comment]
I don't know the inner workings of some of these methods, but it looks like `TEXT.build_vocab` is expecting tokens of the type `string` (which has a `strip` function), not `spacy.tokens.token.Token` which is what spaCy's `Tokenizer` produces. You'll have to make sure the types match when connecting these libraries.

In general, these types of usage questions are probably better asked at a forum like Stackoverflow, where you can get general coding advice from a larger community. We try to keep this issue tracker specifically focused on bugs and feature requests for spaCy. Thanks!
